src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -1006,6 +1005,7 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
     val editModeJsContent = templateResource(clusterResourcesConfig.editModeJs, replacements)
     val safeModeJsContent = templateResource(clusterResourcesConfig.safeModeJs, replacements)
     val jupyterNotebookConfigContent = templateResource(clusterResourcesConfig.jupyterNotebookConfigUri, replacements)
+    val jupyterDockerComposeContent = templateResourceWithNoQuotes(clusterResourcesConfig.jupyterDockerCompose, replacements)
```

Comment:
Again this is probably not needed if you just use the EV in `jupyter-docker-compose.yaml`